packages/ember-toucan-core/src/components/form/controls/character-count.ts

```diff
@@ -0,0 +1,48 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export interface ToucanFormControlCharacterCountComponentSignature {
+  Element: HTMLElement;
```

Comment:
We'll want this to match the `<span` tag so:

```suggestion
  Element: HTMLSpanElement;
```